root/static/scripts/edit/URLCleanup.js

```diff
@@ -2554,16 +2606,43 @@ const CLEANUPS = {
       new RegExp('^(https?://)?([^/]+\\.)?muzikum\\.eu', 'i'),
       new RegExp('^(https?://)?([^/]+\\.)?gutenberg\\.org', 'i'),
     ],
-    type: LINK_TYPES.lyrics,
+    types: [LINK_TYPES.lyrics],
   },
   'mainlynorfolk': {
     match: [new RegExp('^(https?://)?(www\\.)?mainlynorfolk\\.info', 'i')],
-    type: LINK_TYPES.otherdatabases,
+    types: [
+      LINK_TYPES.otherdatabases,
+      {work: [LINK_TYPES.otherdatabases.work, LINK_TYPES.lyrics.work]},
+    ],
```

Comment:
We need it so that these 2 options are shown in the type selector.

Reply:
Is a type selector needed here? I understood that all Mainly Norfolk song pages have lyrics. Am I missing something?

Or do you mean `types` values will always be redundant with `select`results?

Reply:
It's related to the UI, since `types` control what options are shown in the select dropdown, otherwise the type select will show nothing after auto-selection. And yes, `types` values will always be redundant with `select` results?

Reply:
Actually not all Mainly Norfolk song pages have lyrics, so “lyrics” relationship is allowed as an additional relationship to “other databases”. This has been fixed with commits https://github.com/metabrainz/musicbrainz-server/pull/2184/commits/895dd1183d1cb35dd75a52f392f6251509cd8233 and <https://github.com/metabrainz/musicbrainz-server/pull/2184/commits/b436019f728aee0a0e1f52f3b971c7c420809194>.